Index: app/BUILD/src/DeskPRO/Bundle/ImportBundle/Command/ImportCommand.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/BUILD/src/DeskPRO/Bundle/ImportBundle/Command/ImportCommand.php	(revision efe383bb7042efd6fd17394127322f86d298a7f8)
+++ app/BUILD/src/DeskPRO/Bundle/ImportBundle/Command/ImportCommand.php	(revision c79b21e1c8c4ee331d0225022692c47ecb51d13c)
@@ -76,9 +76,12 @@
         $sourceScript = $sourceResolver->getSourceScript($filename, $config);
 
         try {
+            $importedSteps   = $job ? $job->getDataKey('imported_steps', []) : [];
+            $importedOffsets = $job ? $job->getDataKey('import_offsets', []) : [];
+
             $sourceScript->runImport(
-                $job->getDataKey('imported_steps', []),
-                $job->getDataKey('import_offsets', [])
+                $importedSteps,
+                $importedOffsets
             );
             $container->get('dp.importer.source.helper.progress')->finishImport();
         } catch (\Exception $exception) {
